client/modules/i18n/helpers.js

```diff
@@ -45,7 +45,7 @@ Template.registerHelper("currencySymbol", function () {
   });
   const profileCurrency = user.profile && user.profile.currency;
   if (profileCurrency) {
-    const shop = Shops.findOne();
+    const shop = Reaction.getPrimaryShop();
```

Comment:
not positive this should be `getPrimaryShop`, but for now the end result is the same as it was